trade-finance-manager-ui/server/controllers/case/amendments/amendmentAnswers.controller.js

```diff
@@ -64,7 +67,9 @@ const postAmendmentAnswers = async (req, res) => {
       sendFirstTaskEmail: true,
     };
 
-    if (isTfmFacilityEndDateFeatureFlagEnabled()) {
+    const isFacilityEndDateEnabled = isTfmFacilityEndDateFeatureFlagEnabled() && facility.facilitySnapshot.isGef;
+
+    if (isFacilityEndDateEnabled) {
```

Comment:
Yeah this is exactly where we cover it -- There's a rather large amount of tech debt in relation to these tests (and others in this service/codebase) where they are half unit and half api tests. I'm happy this is covering what we want for now, and keen to avoid scope creep to refactor these as they'll take a while